packages/pf4-component-mapper/src/time-picker/time-picker.js

```diff
@@ -19,7 +19,7 @@ const TimePicker = (props) => {
       id={id || input.name}
       FormGroupProps={FormGroupProps}
     >
-      <PF4TimePicker {...input} defaultTime={input.value} {...rest} id={id || input.name} isDisabled={isDisabled || isReadOnly} />
+      <PF4TimePicker {...input} time={input.value ? input.value : undefined} {...rest} id={id || input.name} isDisabled={isDisabled || isReadOnly} />
```

Comment:
I'm not terribly happy with how this component is behaving on the demo page - the input validation happens on every keystroke which prevents you from manually changing `AM` to `PM` or vice versa. Also for some reason the manual entry will auto-complete to AM or PM seemingly randomly if you clear the entire input and try typing `9:00` or similar). To be followed up on this in a separate issue.

Reply:
The pf time picker is a bit clunky from our experience. Mind opening an issue to dig a bit deeper? Karel offered some resources so this might be something they could pick up.

Reply:
@Hyperkid123 apologies for the delay. Here is that issue: https://issues.redhat.com/browse/RHCLOUD-28323. CC @karelhala